controllers/chat/deleteMessage.js

```diff
@@ -0,0 +1,31 @@
+const {StreamChat} = require('stream-chat');
+
+const deleteMessage = async (req, res) => {
+  const client = new StreamChat(process.env.API_KEY, process.env.SECRET_KEY);
+  const {messageID} = req.params;
+
+  try {
+    await client.connectUser(
+      {
+        id: req.userId
+      },
+      req.token
+    );
+
+    let destroy = await client.deleteMessage(messageID, true);
```

Comment:
I think it is safer to change the type of the message to `deleted`
So when the get all messages endpoint is called, the frontend team can still process the deleted message and change the text to `This message is deleted`

Reply:
ok, sudah dirubah jadi soft delete dan otomatis messagenay terganti di getstream